CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ if(Libmultiprocess_ENABLE_CLANG_TIDY)
   set(CMAKE_CXX_CLANG_TIDY "${CLANG_TIDY_EXECUTABLE}")
 endif()
 
+set(EXTERNAL_MPGEN "" CACHE STRING "Use the supplied mpgen binary rather than the one built internally")
```

Comment:
1. The value of `EXTERNAL_MPGEN` must be a full path to satisfy the `if(EXIST...)` condition. Why not make this more explicit by using the `FILEPATH` type and mentioning in the help string?

2. As a user-facing cache variable, it should have a library-specific prefix, like this one:https://github.com/chaincodelabs/libmultiprocess/blob/1e06ff07cdeadc78b2c022750f7b5cbc88d56b1d/CMakeLists.txt#L18

3. style nit: A full stop at the end of the help string?